deploy/cloudformation/manifest-pipeline.yml

```diff
@@ -892,13 +892,6 @@ Resources:
       Timeout: 900
       CodeUri: museum_export/
       Policies:
-        - Version: 2012-10-17
```

Comment:
This applies to all the places where this block was removed: does this function no longer need to execute or read the state machine after its rename? I imagine there's code elsewhere I am missing/didn't review that fills this gap?

Reply:
From @SteveMattison: 
```Removing the 'states:ListStateMachines' and 'states:StartExecution' is unrelated to the renaming, and was just to clean up the code.

When I originally created the first lambda in the series months ago, I copied a block of code defining a lambda from another blueprint.  And then I just used that as a pattern for each of the harvest lambdas.

Ryan pointed out in his review of adding this new lambda that it didn't need 'states:ListStateMachines' or 'states:StartExecution', since it wasn't interacting with the state machine.  So, I removed that from this lambda.

Since this lambda does similar things as the other harvest lambdas, if it wasn't needed here, it shouldn't be needed there, so I removed it from each of the other harvest lambdas.

I'm assuming the state machine executes the lambdas, and the lambdas don't actually do anything with the state machine.```